packages/edit-site/src/components/editor/global-styles-renderer.js

```diff
@@ -100,6 +100,33 @@ export default ( blockData, baseTree, userTree ) => {
 		);
 	};
 
+	const flattenTree = ( input, prefix, token ) => {
```

Comment:
Do we need to render the custom variables on the client? There is no UI to change the custom variables, so I guess we can just keep the ones set on the server?

Reply:
:+1: I'll address in a follow-up PR.